HARK/cstwMPC/cstwMPC.py

```diff
@@ -16,7 +16,7 @@
                            getPercentiles, getLorenzShares, calcSubpopAvg, approxLognormal
 from HARK.simulation import drawDiscrete
 from HARK import Market
-import HARK.cstwMPC.SetupParamsCSTW as Params
+import SetupParamsCSTW as Params
```

Comment:
@sbenthall this is the issue here for https://github.com/econ-ark/DemARK/issues/83